modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc

```diff
@@ -39,14 +39,14 @@ The Chrome extension for Serverless Workflow editor is successfully installed.
 --
 
 [[proc-using-chrome-extension-sw-editor]]
-== Using the Chrome GitHub extension for Serverless Workflow editor
+== Using the Chrome extension for Serverless Workflow editor on GitHub
 
 After installing the Chrome extension for the Serverless Workflow editor, you can use the editor to edit and view the workflow files in GitHub.
 
 .Prerequisites
 * You have installed the Serverless Workflow editor extension in Chrome. 
 +
-For more information, see <<proc-install-chrome-extension-sw-editor, Installing the Chrome GitHub extension for Serverless Workflow editor>>.
+For more information, see <<proc-install-chrome-extension-sw-editor, Installing the Chrome extension for Serverless Workflow editor on GitHub>>.
 
 .Procedure
```

Comment:
Is it possible to update the image `image::tooling/serverless-workflow-editor/swf-editor-in-github.png` to [this one](https://github.com/apache/incubator-kie-kogito-docs/blob/main/serverlessworkflow/modules/ROOT/assets/images/tooling/serverless-workflow-editor/swf-editor-in-github-readonly.png)?

Reply:
Done!